core/templates/domain/topic/editable-topic-backend-api.service.spec.ts

```diff
@@ -16,110 +16,101 @@
  * @fileoverview Unit tests for EditableTopicBackendApiService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-import { TranslatorProviderForTests } from 'tests/test.extras';
-
-require('domain/topic/editable-topic-backend-api.service.ts');
-require('services/csrf-token.service.ts');
-
-describe('Editable topic backend API service', function() {
-  var EditableTopicBackendApiService = null;
-  var sampleDataResults = null;
-  var $httpBackend = null;
-  var CsrfService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(
-    angular.mock.module('oppia', TranslatorProviderForTests));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+import { EditableTopicBackendApiService } from
+  'domain/topic/editable-topic-backend-api.service.ts';
+import { CsrfTokenService } from 'services/csrf-token.service.ts';
+
+describe('Editable topic backend API service', () => {
+  let CsrfService: CsrfTokenService = null;
```

Comment:
The convention that we generally use is ```serviceName: ServiceName``` so it'd be better if this is 
```csrfTokenService: CsrfTokenService```

Reply:
hey @Showtim3 thanks! this is done